src/autocomplete/index.ts

```diff
@@ -91,14 +93,16 @@ class Autocomplete {
     if (activate) {
       this.state.selected = index;
       if (indexExists) {
-        this.updateProducts(target);
+        this.updateProducts(target, updateQuery);
```

Comment:
Maybe you can take the `this.flux.emit('quey:update', query);` and put it above this line, rather than passing through updateQuery. You would then want to pass through refinement, query, and field to updateProducts rather than target probably.

Reply:
That would require destructuring target within setActivation, which may be messier than the current solution, if that's fine then I can do that.

Reply:
Why don't you take `const updateQuery = !!this.config.autocomplete.hoverAutoFill;` just in updateProducts since you are only using that value there?

Reply:
ok Bruce answered it in person with me. It is fine to leave it like this